app/pandas.ipynb

```diff
@@ -535,8 +1101,33 @@
     "%time unknown_signature()"
```

Comment:
The Python solution for calculating the moving average is correct but could be optimized for better performance. Consider using list comprehensions or numpy functions to improve efficiency.